config/locales/en/notifiers/course/assessment/answer/comment_notifier/replied/user_notifications/email.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  notifiers:
+    course:
+      assessment:
+        answer:
+          comment_notifier:
+            replied:
+              user_notifications:
+                email:
+                  subject: '%{course} New comment on %{topic}'
+                  message: >
+                    %{post_author} commented, %{post}
```

Comment:
It just occurred to me that text is HTML so you must need to handle that. Have you tested those?